src/components/home/sprint-view-card.tsx

```diff
@@ -19,17 +23,17 @@ const SprintViewCard = () => {
           >
             {adminMode ? (
               <CardContent>
-              <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
-              {strings.sprint.sprintview}
-              </Typography>
-              <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
-              </CardContent>
-            ) : (
-                <CardContent>
                 <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
-                {strings.sprint.sprintview}
+                  {strings.sprint.sprintview}
                 </Typography>
                 <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
+              </CardContent>
+            ) : (
+                <CardContent>
+                  <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
+                    {strings.sprint.sprintview}
+                  </Typography>
+                  <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
                 </CardContent>
```

Comment:
This should be indented the same amount at the condition above- so indented 2 spaces less.